### PR TITLE
chore: Fix signIn actor

### PR DESCRIPTION
**Pull request by ioanabrooks**
#### Description of changes

* Removed superfluous comments
* Fixes signIn getCurrentUserResolved using event for logging without defining it.

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by thaddmt — APPROVED** [review]

File: packages/ui/src/machines/authenticator/actors/signIn.ts [hunk 1]
```diff
@@ -841,7 +765,7 @@ export function signInActor({ services }: SignInMachineOptions) {
           );
         },
         async getCurrentUserResolved(context) {
-          groupLog('+++getCurrentUserResolved', context, event);
+          groupLog('+++getCurrentUserResolved', context);
```
Comment (calebpollman) [inline packages/ui/src/machines/authenticator/actors/signIn.ts:768]:
```suggestion
```

**Review by dbanksdesign — APPROVED** [review]

File: packages/ui/src/machines/authenticator/actors/signIn.ts (hunk 1 above)
Comment (ioanabrooks) [reply packages/ui/src/machines/authenticator/actors/signIn.ts:768]: Let's remove these logs after we're stable.

Comment (calebpollman) [reply packages/ui/src/machines/authenticator/actors/signIn.ts:768]: Same could be said for removing the commented code that was left to give us a better understanding of the code we are adding?

Comment (ioanabrooks) [reply packages/ui/src/machines/authenticator/actors/signIn.ts:768]: Only removed commented out code that already had implementation in place and I have tested. Additionally we have a ref of that on main if need be.

Comment (calebpollman) [reply packages/ui/src/machines/authenticator/actors/signIn.ts:768]: Just looking for consistency, please choose one or the other. Do not care which

**Pull request merged**